README.md

```diff
@@ -1,6 +1,10 @@
 # PyoMyo
 Python module for the Thalmic Labs Myo armband. 
 
+Adding additional information here.
+
+Providing more context on the frameworks.
+
```

Comment:
The prime directive of line-wrapping is: prefer to break at a higher syntactic level.

When a line is broken at a non-assignment operator the break comes before the symbol. This also applies to the following ...

📄 [Awesomeopensource](https://awesomeopensource.com/project/xmartlabs/Swift-Style-Guide#user-content-where-to-break)

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
The tool supports a number of subcommands, each of which has its own options and are described below. Descriptions of the subcommands that are available can also be obtained by running swift-format --help, an...

📄 [Apple](https://github.com/apple/swift-format#command-line-usage)

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
needs a fix - semicolons
https://www.loom.com/share/69375d2fc30e422cbc69474962773aa7

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix